go/vt/topo/consultopo/watch.go

```diff
@@ -18,19 +18,38 @@ package consultopo
 
 import (
 	"context"
-	"flag"
 	"path"
 	"time"
 
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
+
 	"github.com/hashicorp/consul/api"
 
 	"vitess.io/vitess/go/vt/topo"
```

Comment:
nit: can you fix these up? (`hashicorp/consul` should be up with `spf13/pflag`, `servenv` and `topo` should be in a block together below that)